### ISSUE TITLE
Avoid use of caniuse

### ISSUE THREAD

**Issue by jpmedley** [GoogleChrome/web.dev#1157]
**Describe the bug**
The style guide and the site in general should be cautious of references to caniuse. There's no doubt that it's interface is well done and much liked. Because it is the work of a single individual making it a single point of failure for all pages that reference it. I would suggest the browser compatibility tables on MDN instead.

**Comment by robdodson** [comment]
I think I'm ok with this. @jpmedley I know it's a bit of a chore but would you want to go through and update the stuff that's currently using caniuse and switch it to MDN? I think the main thing would be to make sure the MDN tables are up to date as well.

**Comment by jpmedley** [comment]
Yes. I'm always intent on doing this.

**Comment by jpmedley** [comment]
Clarification: I mean that I already devote time to making sure MDN tables are up to date.

**Comment by robdodson** [comment]
@jpmedley got it. Do you have cycles to look through the docs on web.dev and switch them over to using MDN tables?

**Comment by jpmedley** [comment]
I've put it on my list. Regardless of whether of caniuse or MDN is used, we need to make sure data in MDN is up to date. We need some kind of process for making sure that's checked.

**Comment by jpmedley** [comment]
Here's the list of articles that currently mention caniuse. I need to make sure MDN is correct before I change any article. If one of these gets updated, I'll prioritize it over the others. 

.//lighthouse-performance/uses-webp-images/index.md
.//lighthouse-performance/uses-rel-preload/index.md
.//fast/serve-modern-code-to-modern-browsers/index.md
.//fast/avoid-invisible-text/index.md
.//fast/codelab-serve-modern-code/index.md
.//fast/codelab-preload-critical-assets/index.md
.//fast/serve-images-webp/index.md
.//fast/codelab-text-compression-brotli/index.md
.//fast/adaptive-serving-based-on-network-quality/index.md
.//fast/serve-responsive-images/index.md
.//fast/codelab-text-compression/index.md
.//blog/javascript-and-google-search-io-2019/index.md
.//blog/prefers-color-scheme/index.md
.//blog/use-srcset-to-automatically-choose-the-right-image/index.md

**Comment by jpmedley** [comment]
Here's some recent news from MDN: 

[Caniuse and MDN compatibility data collaboration](https://hacks.mozilla.org/2019/09/caniuse-and-mdn-compat-data-collaboration/)

caniuse now has a script that will convert MDN data on the fly and show it in it's own interface. The importance of this can't be understated. The money quote is this:

>The caniuse support table count expands from roughly 500 to 10,500 tables!

I knew there was a difference in the amount of data between the two. I did't know it was that large. 

>MDN’s support data is very broad and covers feature support at a very granular level. This allows MDN to provide as much detailed information as possible across all web technologies, supplementing the reference information provided by MDN Web Docs.

The bottom line is that MDN *requires* more data than caniuse. And now some version of that will appear on caniuse.

**Comment by jpmedley** [comment]
I should still check that browser data is correct on web.dev pages. I've added this to my to-do list.
